<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabConfigWg</class>
 <widget class="QWidget" name="TabConfigWg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="category_gbx">
     <property name="title">
      <string>Category</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="config_category_group_layout"/>
    </widget>
   </item>
   <item>
    <widget class="InfoWidgets" name="config_tb" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoWidgets</class>
   <extends>QWidget</extends>
   <header location="global">widgets/infowidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
